storybook/main.qml

```diff
@@ -140,6 +140,20 @@ ApplicationWindow {
                         onClicked: settingsPopup.open()
                     }
 
+                    CheckBox {
+                        id: windowAlwaysOnTopCheckBox
+
+                        Layout.fillWidth: true
+
+                        text: "Always on top"
```

Comment:
Um, why is this needed? :thinking:

Reply:
Well, sometimes it's good to have it on top while you're live-coding.
I saw @micieslak using this option time to time, but he uses OS instruments to do this. And MacOS doesn't have such feature 🤷‍♂️